easybuild/easyconfigs/o/OpenMM/OpenMM-7.7.0-foss-2021a.eb

```diff
@@ -0,0 +1,69 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+# Update to 7.5.1
+# J. Sassmannshausen / GSTT
+
+easyblock = 'CMakeMake'
+
+name = 'OpenMM'
+version = '7.7.0'
```

Comment:
I'm checking with other maintainers to be sure, but now that we don't use the fosscuda toolchain anymore, I think we always put a `versionsuffix = '-CUDA-%(cudaver)s'`. But as said, I'm still checking, I'll let you know once I'm sure.

Reply:
You are right, once I put this eb in production I realized I forgot about `versionsuffix = '-CUDA-%(cudaver)s'!